cmd/root.go

```diff
@@ -123,13 +123,12 @@ func initConfig() {
 				fmt.Println(err)
 				os.Exit(1)
 			}
-
 			viper.AddConfigPath(".")
 			viper.AddConfigPath(home)
 			viper.SetConfigName("kubetel")
 		}
-		viper.AutomaticEnv()
-		viper.SetEnvPrefix("KUBETEL")
+		//viper.AutomaticEnv()
```

Comment:
why did you comment it out? it basically disable reading config from env

Reply:
Commented back, this is a residue on local testing, lol